app/graphql/mutations/post/unlock_post.rb

```diff
@@ -0,0 +1,31 @@
+class Mutations::Post::UnlockPost < Mutations::Base
+  argument :input,
+    Types::Input::Post::Unlock,
+    required: true,
+    description: 'Unlock a Post.',
+    as: :post
+
+  field :post, Types::Post, null: true
+
+  def load_post(value)
+    post = ::Post.find(value.id)
+    post.assign_attributes(value.to_model)
+    post
```

Comment:
I'm distinctly not a fan of this thing where you get the changes from some other file and then shove them in over here... It seems like indirection without purpose

Reply:
Funny enough this is how it was originally done (by Matt). I liked it so I kinda just started using it also lol.